### PR TITLE
Submitting my details 

### PR DESCRIPTION
**Pull request by saks1209am**
# Application to the Class of 2020🎓

This pull request template helps you complete an application to the **Class of 2020**. Use the checklist below to verify you have followed the instructions correctly. 

## Checklist ✅

- [x] I have read the instructions on the README file before submitting my application. 
- [x] I made my submission by creating a folder on the `_data` folder and followed the naming convention mentioned in the instructions (`<username>`), added my profile picture and markdown file.
- [x] I have used the Markdown file template to add my information to the Year Book.
- [x] I have completed the form: `https://airtable.com/shrmuHxu38ZariKJi`
- [x] My application belongs to:
  - [x] **Tier 1** 📖: I added myself to the Year Book.
  - [ ] **Tier 2** ✏️: I wrote a post on DEV about a project I built.
  - [ ] **Tier 3** ✨: I want my project to be highlighted on stream.
- [x] I understand that a reviewer will merge my pull request after examining it or ask for changes in case needed.
- [x] I understand I should not tag or add a reviewer to this Pull Request.
- [x] I understand the photo added to the template will be used in the ceremony "Graduate Walk". 
- [x] I have [added the event](http://www.google.com/calendar/event?action=TEMPLATE&dates=20200615T160000Z%2F20200615T183000Z&text=%24%20git%20remote%20%3Cgraduation%3E%20%F0%9F%8E%93&location=https%3A%2F%2Fwww.twitch.tv%2Fgithubeducation&details=) to my Calendar.

### CODE REVIEW

**Comment by saks1209am** [comment]
Can you help me? I didn't get what you are saying

**Comment by darkRaspberry** [comment]
In the checklist which u filled during creating this PR. Your first comment and edit that comment.

And 

Just add the "x" (small x) between [] ( square bracket)
like this [x] not this [ x] or [x ] or [ ] x

only Correct way [x]

**Comment by saks1209am** [comment]
Now I have just selected those checkboxes which tasks I have done. I didn't do any [x] in that. I would appreciate it if you can help me in creating my first PR for this repository.

**Comment by darkRaspberry** [comment]
Thats nice. Wait till merge

**Comment by saks1209am** [comment]
Bro, I have a query.  In my  .jpg file I have passed my Image URL but it is not displaying my image in the preview section. Can you solve this issue?

File: _data/saks1209am/saksham.jpg [hunk 1]
```diff
@@ -0,0 +1 @@
+![Saksham](https://user-images.githubusercontent.com/30996074/82549799-2149f880-9b7b-11ea-85dc-959722465e91.JPG)
```
Comment (darkRaspberry) [inline _data/saks1209am/saksham.jpg:1]:
```suggestion
Put the whole file name as `Saksham.jpg`
```

No you don't have to pass any url. Just add the image in the folder and commit.

Comment (darkRaspberry) [reply _data/saks1209am/saksham.jpg:1]: Simply delete this img file and paste a original `Saksham.jpg` file which is your picture.

Comment (saks1209am) [reply _data/saks1209am/saksham.jpg]: Did it bro. Do i again need to create a new PR?

**Comment by darkRaspberry** [comment]
Wait till merged its good

**Comment by saks1209am** [comment]
So, should I edit my .jpg file and create a new pull request for it.

**Comment by darkRaspberry** [comment]
Your image is too small , Add a new image in this PR. which is minimum 544x544.
And reopen this PR.

**Comment by saks1209am** [comment]
How can create reopen this PR?. Should I create a new one or can you guide me how to reopen this PR as it is not closed by me.

**Comment by darkRaspberry** [comment]
just update the new square image and commit in your fork
and it should here. Then just reopen this PR

**Comment by saks1209am** [comment]
I have updated my image in the fork but when I'm clicking on pull request it is creating a new PR for me

**Comment by saks1209am** [comment]
I have made the suggested changes, what do I need to do now. I am not getting how to reopen this pull request.

**Comment by saks1209am** [comment]
Hey @darkRaspberry can you please guide me on how can I reopen my pull request. I have made changes in my fork but don't know how to reopen this PR again

**Pull request closed without merging**